kernelci.org/content/en/docs/instances/local.md

```diff
@@ -0,0 +1,232 @@
+        # If set to the master-key this field will have to be updated
+        # once the tokens created
+        backend_token:
+            "<random_string>"
+        # A secret key internally used by Flask
+        secret_key:
+            "<random_string>"
+        # The url location of the storage server for the backend
+        file_server:
+            "http://kci-vm"
+        ```
+
+        ```
+        ssl_stapling_resolver:
+            "localhost"
+        ```
+
+
+> You can use UUID as your random string. You can easily generate one with
+> ```
+> python -c "import uuid;print(uuid.uuid4())"
+> ```
+
+- Run ansible playbook to set up your VM
+
+        ```
+        ansible-playbook -i hosts -l kci-vm -D -b -K  -e git_head="main" -e "@./dev-secrets.yml" --key-file ~/.ssh/org/id_rsa dev.yml
+        ```
+
+> **Note** The nginx configuration step may fail due to SSL issues, but that's fine for the development setup.
+
+- Tweak your VM config
+
+    - Log in to your VM as root
+    - Delete backend nginx config
+
+    ```
+    rm /etc/nginx/sites-enabled/kci-vm
+    ```
+
+    - Replace content of the storage config (`/etc/nginx/sites-enabled/kci-vm.conf`)
+
+    ```
+    server {
+        listen *;
+        listen [::];
+
+        server_name kci-vm;
+        root /var/www/images/kernel-ci;
+        charset utf-8;
+
+        access_log /var/log/nginx/kci-vm-access.log combined buffer=16k;
+        error_log /var/log/nginx/kci-vm-error.log crit;
+
+        location / {
+            if (-f $document_root/maintenance.html) {
+                return 503;
+            }
+
+            fancyindex on;
+            fancyindex_exact_size off;
+        }
+    }
+
+    ```
+
+    - Restart nginx
+
+    ```
+    systemctl restart nginx
+    ```
+
+    - Start KernelCI services
+
+    ```
+    systemctl start kernelci-backend
+    systemctl start kernelci-celery
+    ```
+
+> Note
+> At this point your kernelci-backend instance should be available at `http://kci-vm:8888`
+
+> Note
+> If you want to follow your kenrelci logs use journalctl
+> ```
+> journalctl -f -u kernelci-celery -u kernelci-backend
+> ```
+
+## Configure KernelCI Backend
+
+As the instance of `kernelci-backend` is up and running you can add necessary configuration to it.
+
+- Create admin token
+
+    ```
+    curl -XPOST -H "Content-Type: application/json" -H "Authorization: YOUR_MASTER_KEY" "http://kci-vm:8888/token" -d '{"email": "me@example.com", "username": "admin", "admin": 1}'
+    ```
+
+    If this goes well, you should see the admin token in the output
+    e.g.
+
+    ```
+    {"result":[{"token":"3f1fbc0f-4146-408c-a2da-02748f595bfe"}],"code":201}
+    ```
+
+## Build kernel and push results
+
+Congratulations, your `kernelci-backend` instance is up and running.
+You can now build your kernel and push the artifacts with `kci_build` and `kci_data`.
+See the [`kci_build`](https://github.com/kernelci/kernelci-core/blob/main/kci_build) documentation to get you started.
```

Comment:
This points to source code not `kci_build` documentation

Reply:
good point. Fixed.